klarna-on-demand/build.gradle

```diff
@@ -1,4 +1,6 @@
 apply plugin: 'com.android.library'
+apply plugin: 'maven'
+apply plugin: 'signing'
 
 android {
```

Comment:
Have you considered using this gradle-mvn-push.gradle:
apply from: 'https://raw.github.com/shamanland/gradle-mvn-push/cc18d56549cdea03f744b6fff27911569394073e/gradle-mvn-push.gradle'

Then we only need to configure stuff in the gradle.property file